v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/options/Neo4jFlexTemplateOptions.java

```diff
@@ -60,9 +61,10 @@ public interface Neo4jFlexTemplateOptions extends CommonTemplateOptions {
   @TemplateParameter.Text(
       order = 4,
       optional = true,
-      description = "Options JSON",
-      helpText = "Options JSON. Use runtime tokens.",
-      example = "{token1:value1,token2:value2}")
+      description =
+          "JSON object also called \"runtime tokens\", defining named values interpolated into the specification URIs and queries.",
+      helpText = "JSON object also called \"runtime tokens\"",
+      example = "{token1:value1,token2:value2}. Spec can refer to $token1 and $token2.")
```

Comment:
What does "Spec" mean here?

Reply:
sorry, it's the import specification, whose URI is specified with the `jobSpecUri` option